### ISSUE TITLE
ERROR [backup.ui.uiserver] 

### ISSUE THREAD

**Issue by PrathikGopal** [sabeechen/hassio-google-drive-backup#267]
###### Description:
```
First of all a big thank-you to you, i really appreciate what you do for the community. This add-on is really important to me and its working. As of recent update i noticed few errors continuously popping up. Though Backup is working as intended, but there is a bunch of errors on the supervisor and the addon logs about http errors. Could you help check this?
```

 Addon version: `0.101.3`
 Home Assistant Version: `0.115.3`
 Supervisor Version: `245`
 Supervisor Channel: `stable`
 Hassos Version: `4.13`
 Docker Version: `19.03.11`
 Architecture: `armv7`
 Machine: `raspberrypi4`
 Date: `2020-09-26T09:56:14.576336+00:00`
 Timezone: `Asia/Kolkata`
 Failure Time: `Never`
 Last Good Sync: `2020-09-26T09:47:56.052571+00:00`
 ###### Exception:
 ```
 No error could be identified automatically.
 ```
 Snapshots:
 ```
 {
    "HomeAssistant": {
        "snapshots": 3,
        "retained": 0,
        "deletable": 3,
        "name": "HomeAssistant",
        "latest": "2020-09-25T23:09:36Z",
        "size": "1.4 GB"
    },
    "GoogleDrive": {
        "snapshots": 1,
        "retained": 0,
        "deletable": 1,
        "name": "GoogleDrive",
        "latest": "2020-09-25T23:09:36Z",
        "size": "487.8 MB"
    }
}
 ```
 ###### Config:
 ```
 {
    "Setting.MAX_SNAPSHOTS_IN_HASSIO": 3,
    "Setting.MAX_SNAPSHOTS_IN_GOOGLE_DRIVE": 1,
    "Setting.DAYS_BETWEEN_SNAPSHOTS": 2.0,
    "Setting.SNAPSHOT_TIME_OF_DAY": "04:00",
    "Setting.SNAPSHOT_PASSWORD": "REDACTED",
    "Setting.REQUIRE_LOGIN": true,
    "Setting.EXPOSE_EXTRA_SERVER": true,
    "Setting.GOOGLE_DRIVE_TIMEOUT_SECONDS": 320.0,
    "Setting.ALTERNATE_DNS_SERVERS": "8.8.8.8,1.1.1.1"
}
 ```
 ###### Addon Logs:
 ```
 09-26 15:19:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:19:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:19:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:19:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:20:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:20:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:20:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:20:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:21:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:21:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:21:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:21:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:22:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:22:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:22:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:22:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:23:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:23:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:23:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:23:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:24:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:24:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:24:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:24:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:25:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:25:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:25:43 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:25:43 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

09-26 15:26:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/auth
09-26 15:26:13 ERROR [backup.ui.uiserver] 
  addon/backup/ui/uiserver.py:745 (check_credentials)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:60 (auth)
  addon/backup/ha/harequests.py:26 (wrap_and_call)
  addon/backup/ha/harequests.py:213 (_postHassioData)
  addon/backup/ha/harequests.py:175 (_validateHassioReply)
  /site-packages/aiohttp/client_reqrep.py:941 (raise_for_status)
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://hassio/auth')

 ```
 ###### Supervisor Logs:
 ```
   File "/usr/local/lib/python3.8/site-packages/aiohttp/helpers.py", line 145, in decode
    raise ValueError('Unknown authorization method %s' % auth_type)
ValueError: Unknown authorization method Bearer[0m
[31m20-09-26 09:56:13 ERROR (MainThread) [aiohttp.server] Error handling request[0m
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_protocol.py", line 418, in start
    resp = await task
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/integrations/aiohttp.py", line 110, in sentry_app_handle
    reraise(*_capture_exception(hub))
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/_compat.py", line 55, in reraise
    raise value
  File "/usr/local/lib/python3.8/site-packages/sentry_sdk/integrations/aiohttp.py", line 100, in sentry_app_handle
    response = await old_handle(self, request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_app.py", line 458, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.8/site-packages/aiohttp/web_middlewares.py", line 119, in impl
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/security.py", line 172, in token_validation
    return await handler(request)
  File "/usr/src/supervisor/supervisor/api/utils.py", line 59, in wrap_api
    answer = await method(api, *args, **kwargs)
  File "/usr/src/supervisor/supervisor/api/auth.py", line 66, in auth
    return await self._process_basic(request, addon)
  File "/usr/src/supervisor/supervisor/api/auth.py", line 41, in _process_basic
    auth = BasicAuth.decode(request.headers[AUTHORIZATION])
  File "/usr/local/lib/python3.8/site-packages/aiohttp/helpers.py", line 145, in decode
    raise ValueError('Unknown authorization method %s' % auth_type)
ValueError: Unknown authorization method Bearer[0m
[32m20-09-26 09:56:14 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m

 ```
 ###### Home Assistant Core Logs:
 ```
 2020-09-26 15:24:56 ERROR (MainThread) [homeassistant.components.plex] Plex server (https://192-168-0-102.4d896d2709004a1393a0fc9e9fd2b016.plex.direct:32400) could not be reached: [HTTPSConnectionPool(host='192-168-0-102.4d896d2709004a1393a0fc9e9fd2b016.plex.direct', port=32400): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x9bea2a30>: Failed to establish a new connection: [Errno 113] Host is unreachable'))]
2020-09-26 15:24:56 WARNING (MainThread) [homeassistant.config_entries] Config entry for plex not ready yet. Retrying in 80 seconds
2020-09-26 15:25:03 WARNING (MainThread) [homeassistant.config_entries] Config entry for sonarr not ready yet. Retrying in 80 seconds
2020-09-26 15:25:24 ERROR (MainThread) [homeassistant.helpers.entity] Update for light.hyperion fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 278, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 471, in async_device_update
    await self.hass.async_add_executor_job(self.update)  # type: ignore
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/hyperion/light.py", line 219, in update
    if "activeLedColor" not in response["info"]:
KeyError: 'info'
2020-09-26 15:25:45 ERROR (MainThread) [homeassistant.components.wled] Error fetching wled data: Invalid response from API: Error occurred while communicating with WLED device at 192.168.0.196
2020-09-26 15:25:45 WARNING (MainThread) [homeassistant.config_entries] Config entry for wled not ready yet. Retrying in 80 seconds
2020-09-26 15:25:54 ERROR (MainThread) [glances_api] Can not load data from Glances API
2020-09-26 15:25:54 WARNING (MainThread) [homeassistant.config_entries] Config entry for glances not ready yet. Retrying in 80 seconds
2020-09-26 15:25:54 ERROR (MainThread) [homeassistant.helpers.entity] Update for light.hyperion fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 278, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 471, in async_device_update
    await self.hass.async_add_executor_job(self.update)  # type: ignore
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/src/homeassistant/homeassistant/components/hyperion/light.py", line 219, in update
    if "activeLedColor" not in response["info"]:
KeyError: 'info'

 ```

**Comment by sabeechen** [comment]
It looks like you have the extra server running, which you can disable in the addon's settings dialog or by removing the `require_login` and `expose_extra_server` keys from your yaml config.  Most people have this running leftover from before the addon supported ingress, at least it sounds like you're already using ingress.

That being said, it looks like something is wrong with username/password auth with the extra server.  I'll need to look into that.

**Comment by sabeechen** [comment]
I just released the addon version 0.102.0, which includes a fix for this.  Please let me know or reopen this issue if you run into any trouble with the new version.
Also just a reminder, to make Home Assistant see the new version right away go to "Supervisor" > "Add-on Store" > Three dots in the top right corner > "Reload"

**Comment by PrathikGopal** [comment]
This is resolved, thank you very much @sabeechen .
